.circleci/config.yml

```diff
@@ -135,28 +144,31 @@ workflows:
           name: build-<< matrix.os >>
           matrix:
             parameters:
-              os: [linux-node15, linux-node10]
+              os: [linux_node15, linux_node10, windows_node12, macos_node12]
       - test:
           name: test-<< matrix.os >>
           matrix:
             parameters:
-              os: [linux-node15, linux-node10]
+              os: [linux_node15, linux_node10, windows_node12, macos_node12]
           requires:
             - build-<< matrix.os >>
       - e2e-test:
           name: e2e-test-<< matrix.os >>
           matrix:
             parameters:
-              os: [linux-node15, linux-node10]
+              os: [linux_node15, linux_node10, macos_node12]
```

Comment:
Do we have any failing tests for windows?

Reply:
The test setup itself was failing on windows. I tried to run the e2e suite on Amazon Workspaces windows machine and found that the `spawn` function from `nexpect` does not work as intended on windows.